.chloggen/sdk-telemetry-span-metrics.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: 'telemetry'
```

Comment:
I think the telemetry.sdk prefix is too broad.  Unless we are developing these with SDKs other than opentelemetry, I would propose using the prefix: `otel.sdk`

Reply:
I think the intention of the meaning of `telemetry` in this case is a different one: It's (self-)telemetry of the `otel.sdk` (not in the sense of "telemetry" being something broader than "otel").

Still, I agree `otel.sdk` should good enough as the namespace, here.

Reply:
Moved to `otel.sdk` in 67ab2d072c228f